pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -422,6 +422,43 @@ task translate_augur_tree {
     }
 }
 
+task assign_clades_to_nodes {
+    meta {
+        description: "Assign taxonomic clades to tree nodes based on mutation information"
+    }
+    input {
+        File tree_nwk
+        File nt_muts_json
+        File aa_muts_json
+        File ref_fasta
+        File clades_tsv
+
+        Int?   machine_mem_gb
+        String docker = "nextstrain/base"
+    }
+    String out_basename = basename(basename(tree_nwk, ".nwk"), "_refined_tree")
+    command {
+        augur clades \
+        --tree ~{tree_nwk} \
+        --mutations ~{nt_muts_json} ~{aa_muts_json} \
+        --reference ~{ref_fasta} \
+        --clades ~{clades_tsv} \
+        --output-node-data ~{out_basename}_node-clade-assignments.json
+    }
+    runtime {
+        docker: docker
+        memory: select_first([machine_mem_gb, 3]) + " GB"
+        cpu :   2
+        disks:  "local-disk 50 HDD"
+        dx_instance_type: "mem1_ssd1_v2_x2"
+        preemptible: 2
+    }
+    output {
+        File node_clade_data_json = "~{out_basename}_node-clade-assignments.json"
+        String augur_version      = read_string("VERSION")
```

Comment:
Doesn't look like you wrote the `VERSION` file in the above command block

Reply:
I don't know how long this task would take with larger datasets so would it be a good idea to parameterize preemptibles and cpu as well?

Reply:
Oh actually I was going to suggest the opposite--can we deparameterize the memory? It's never going to need any more than what we put here. I was thinking of deparameterizing a few of the other cheap steps as well (augur export, traits, etc) while upping the default values on the heavy steps (align, draft, refine) but that's for a separate PR.

Reply:
Yeah there isn't really a right/wrong - I think whatever works best for this specific task.

Reply:
Yeah, this step runs fine with our large SARS-CoV-2 tree in a VM with 4GB of RAM.